sync/docker.ps1

```diff
@@ -12,3 +12,7 @@ Install-Module -Name DockerMsftProvider -Force
 Write-Output "### Installing Docker"
 
 Install-Package Docker -Providername DockerMsftProvider -Force
+
+Write-Output "### Installing HyperV"
+
+Install-WindowsFeature -Name Hyper-V -IncludeManagementTools -Restart
```

Comment:
```
 winw1: Install-WindowsFeature : A prerequisite check for the Hyper-V feature failed.
    winw1: 1. Hyper-V cannot be installed: The processor does not support second level address translation (SLAT).
    winw1: At C:\k\Install-Containerd.ps1:106 char:9
    winw1: +         Install-WindowsFeature -Name $feature
    winw1: +         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    winw1:     + CategoryInfo          : InvalidOperation: (Hyper-V:ServerComponentWrapper) [Install-WindowsFeature], Exception
    winw1:     + FullyQualifiedErrorId : Alteration_PrerequisiteCheck_Failed,Microsoft.Windows.ServerManager.Commands.AddWindowsF 
    winw1:    eatureCommand
    winw1: ### Running PrepareNode.ps1
    winw1: C:\k\PrepareNode.ps1 : ContainerD service was not detected - please install and start containerD before calling 
    winw1: PrepareNode.ps1 with -ContainerRuntime containerD
```
Turns out virtual box does not support nesting of HyperV only LKM and Virtualbox are supported